### PR TITLE
Fixes data-fragment-index on code block

### PR DESCRIPTION
**Pull request by jtama**
Links #532

### CODE REVIEW

File: lib/asciidoctor-revealjs/highlightjs.rb [hunk 1]
```diff
@@ -223,9 +223,11 @@ def docinfo location, doc, opts
         var highlightSteps = RevealHighlight.deserializeHighlightSteps( block.getAttribute( 'data-line-numbers' ) );
         if( highlightSteps.length > 1 ) {
 
+          // When rendered with asciidoc, data-fragment index if it exists lies on the listing block element, so we have to lookup
           // If the original code block has a fragment-index,
           // each clone should follow in an incremental sequence
-          var fragmentIndex = parseInt( block.getAttribute( 'data-fragment-index' ), 10 );
```
Comment (ggrossetie) [inline lib/asciidoctor-revealjs/highlightjs.rb:228]: Could you please add a variable name on each `parentElement`? Something like:

```js
var code = block.parentElement
var pre = code.parentElement
var listingBlock = pre.parentElement
```

**Comment by ggrossetie** [comment]
Thanks!
Could you please update this test `source-highlightjs-languages:source-highlightjs-languages` (see CI logs)

**Comment by jtama** [comment]
Forced push the changes. Tests are passing on my machine 😀

**Comment by jtama** [comment]
Do you want a sample of it in action ?

**Comment by ggrossetie** [comment]
> Forced push the changes. Tests are passing on my machine

And it also works on the CI 😉 

> Do you want a sample of it in action ?

Yes, that would be great! Please add an example in https://github.com/asciidoctor/asciidoctor-reveal.js/blob/master/examples/release-5.2.adoc (it will be published by Netlify: https://deploy-preview-533--asciidoctor-revealjs-examples.netlify.app/release-5.2)

**Comment by jtama** [comment]
Done !

**Comment by ggrossetie** [comment]
That's very cool! 👍🏻 
https://deploy-preview-533--asciidoctor-revealjs-examples.netlify.app/release-5.2#/_synchronising_step_by_step_syntax_highlighting

**Comment by ipocentro87** [comment]
Hi, thanks a lot for the update.

I wanted to make the listing appearing immediately in the slide, together with the first fragment, do you know how to make that work? 
Could it be related to this `fragmentIndex += 1;`? It seems that even if I set `step=0` the listing is not showing.

**Comment by jtama** [comment]
Not sure exactly what you want to achieve.

You want the code to be displayed as soon as the slide is focused with the first callout, is this correct ?

**Comment by ipocentro87** [comment]
Yes, basically I'd like that as soon as you move onto the slide, the code snippet that is within that slide is visible, with the focus on the highlighted line at the first step. 
For instance, I have the following

```
[source,,linenums,highlight="2|3-6|2|3-6|2|3-6|13", step=0]
----
<some code>
----
````

And I'd like that as soon as I jump to that slide, the code snippet is shown, and line 2 is highlighted.

I managed to make a workaround, but it is not ideal as I had to duplicate the code and it doesn't highlight the line 2 straight away.

```
# overlay the two code blocks
[.r-stack] 
--

[source,,linenums,highlight="2|3-6|2|3-6|2|3-6|13", step=0]
----
<some code>
----

[source.fade-out,,linenums,step=0]
----
<some code>
----

--
```

**Pull request merged**